Sources/Components/Components/Onboarding/WKOnboardingCell.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+struct WKOnboardingCell: View {
+
+    // MARK: - Properties
+
+    @ObservedObject var appEnvironment = WKAppEnvironment.current
+
+    var viewModel: WKOnboardingViewModel.WKOnboardingCellViewModel
+
+    // MARK: - Lifecycle
+
+    var body: some View {
+        HStack {
+            VStack {
+                if let icon = viewModel.icon {
+                    Image(uiImage: icon)
+                        .frame(width: 24, alignment: .top)
+                        .accessibilityHidden(true)
+                        .scaledToFill()
+                        .padding([.trailing], 12)
+                }
+                Spacer()
+            }
+            VStack() {
+                Text(viewModel.title)
+                    .multilineTextAlignment(.leading)
+                    .font(WKFont.for(.body).bold())
```

Comment:
I think we should avoid the `bold` modifier and instead try to bury that logic into `WKFont`. Here's a file from @staykids's components example, you can see how the font weight is added in `prominentTitle` and `prominentHeadline`.